.github/workflows/master.yml

```diff
@@ -197,6 +197,12 @@ jobs:
         run: |
           python -c "import mlflow"
 
+          # Just importing mlflow should not create `mlruns` directory
+          # See: https://github.com/mlflow/mlflow/issues/3400
+          if [ -d "./mlruns" ]; then
+            exit 1
+          fi
```

Comment:
@tmcclintock If `python -c "import mlflow"` still created the `mlruns` directory, this check would fail.

Reply:
As I wrote in the issue: I was accidentally had `master` installed when I tried the test. Your fix works. Thanks again.